src/test/kotlin/com/roadlink/tripservice/usecases/trip_search/SearchTripTest.kt

```diff
@@ -30,104 +46,102 @@ internal class SearchTripTest {
 
     @Test
     fun `given no section exists then should return empty list`() {
+        // GIVEN
+        val departure = LocationFactory.avCabildo_4853()
+        val arrival = LocationFactory.avCabildo_20()
+        val at = InstantFactory.october15_12hs()
+        circleSearchAreaRadius = circleSearchAreaRadius(departure, arrival)
+        givenNextSections(from = departure, at = at, nextSections = emptySet())
+
+        // WHEN
         val result = searchTrip(
             SearchTrip.Input(
-            departure = LocationFactory.avCabildo_4853(),
-            arrival = LocationFactory.avCabildo_20(),
-            at = InstantFactory.october15_12hs(),
+            departure = departure,
+            arrival = arrival,
+            at = at,
         ))
 
+        // THEN
         assertTrue(result.isEmpty())
     }
 
     @Test
     fun `given exists a trip plan between the given departure and arrival then should return it`() {
```

Comment:
NIT: given an existing trip plan between a departure and arrival date, then it should be retrive it